### DISCUSSION TITLE
Receive warning "Function components cannot be given refs." with drop zone

### DISCUSSION THREAD

**Discussion by pat-mok** [chakra-ui/chakra-ui#9293] [Q&A] [answered]
I simply copy and paste the drop zone component code from the file upload component doc page from Chakra and received this warning in the console "Function components cannot be given refs". Any suggestion to fix?

**Comment by segunadebayo** [comment]
Do you have a reproduction for this? That feels weird since the refs are forwarded correctly

**Comment by segunadebayo** [comment, marked as answer]
The current snippet forwards the ref. Re-add it and the warning should go away.
